src/App.tsx

```diff
@@ -1,26 +1,32 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useAppSelector } from './app/hooks';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const { loading } = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo) || null;
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  return (
+    <>
+      <div className="section">
+        <div className="container">
+          <div className="box">
+            <h1 className="title">Todos:</h1>
+
+            <div className="block">
+              <TodoFilter />
+            </div>
 
-          <div className="block">
-            <Loader />
-            <TodoList />
+            <div className="block">
+              {loading && <Loader />}
```

Comment:
Consider verifying that the `Loader` component is correctly implemented to display a loading indicator when `loading` is true. Additionally, ensure that the `TodoList` component can handle scenarios where `loading` is true to prevent any rendering issues.